tests/OAuth2SummitAttendeeNotesApiTest.php

```diff
@@ -0,0 +1,198 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+class OAuth2SummitAttendeeNotesApiTest extends ProtectedApiTest
+{
+    use InsertSummitTestData;
+
+    use InsertMemberTestData;
+
+    protected function setUp():void
+    {
+        parent::setUp();
+        self::insertMemberTestData(IGroup::TrackChairs);
+        self::$defaultMember = self::$member;
+        self::$defaultMember2 = self::$member2;
+        self::insertSummitTestData();
+    }
+
+    protected function tearDown():void
+    {
+        self::clearSummitTestData();
+        self::clearMemberTestData();
+        parent::tearDown();
+    }
+
+    public function testGetAttendeeNotes(){
+
+        $params = [
+            'id'          => self::$summit->getId(),
+            'attendee_id' => self::$summit->getAttendees()[0]->getId()
+        ];
+
+        $headers = [
+            "HTTP_Authorization" => " Bearer " . $this->access_token,
+            "CONTENT_TYPE"        => "application/json"
+        ];
+
+        $response = $this->action(
+            "GET",
+            "OAuth2SummitAttendeeNotesApiController@getAttendeeNotes",
+            $params,
+            [],
+            [],
+            [],
+            $headers
+        );
+
+        $content = $response->getContent();
+        $this->assertResponseStatus(200);
+        $attendee_notes = json_decode($content);
+        $this->assertTrue(!is_null($attendee_notes));
+    }
+
+    public function testGetAttendeeNote() {
+        $attendee = self::$summit->getAttendees()[0];
+        $attendee_note = $attendee->getNotes()[0];
+
+        $params = [
+            'id'          => self::$summit->getId(),
+            'attendee_id' => $attendee->getId(),
+            'note_id'     => $attendee_note->getId(),
+            'expand'      => 'owner,author'
+        ];
+
+        $headers = [
+            "HTTP_Authorization" => " Bearer " . $this->access_token,
+            "CONTENT_TYPE"        => "application/json"
+        ];
+
+        $response = $this->action(
+            "GET",
+            "OAuth2SummitAttendeeNotesApiController@getAttendeeNote",
+            $params,
+            [],
+            [],
+            [],
+            $headers
+        );
+
+        $content = $response->getContent();
+        $this->assertResponseStatus(200);
+        $attendee_note = json_decode($content);
+        $this->assertTrue(!is_null($attendee_note));
+    }
+
+    public function testAddAttendeeNote() {
+        $params = [
+            'id'          => self::$summit->getId(),
+            'attendee_id' => self::$summit->getAttendees()[0]->getId(),
+        ];
+
+        $data = [
+            'content'   => 'Test attendee note 2',
+            'ticket_id' => 7778
```

Comment:
does this ticket exists on test data ? just use test data not hardcoded one

Reply:
fixed